visualizer/selection-controls.css

```diff
@@ -9,20 +9,12 @@
 }
 
 #selection-controls .hotness-selector {
-  padding: 4px 8px;
   margin-right: 4px;
-  background-color: var(--opposite-contrast);
-  color: var(--max-contrast);
-  border: none;
-  cursor: pointer;
-  display: flex;
-  align-items: center;
-  line-height: 1;
 }
-#selection-controls button.hotness-selector:hover:not(:disabled) {
+/* #selection-controls button.hotness-selector:hover:not(:disabled) {
```

Comment:
Probably better to just remove these than comment out 👍

Reply:
👍